uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/LeaveRequest.php

```diff
@@ -251,7 +251,7 @@ private static function validateTOILPastDays($params) {
 
     if ($leaveDatesHasPastDates && !$absenceType->allow_accrue_in_the_past) {
       throw new InvalidLeaveRequestException(
-        'You cannot request TOIL for past days',
+        'You may only request TOIL for Overtime to be worked in the future. Please modify the date of this request',
```

Comment:
I'm not sure if I'm reviewing the messages or if they're just copied and pasted, but not sure why overtime is capitalized here.